docs/core/windows-prerequisites.md

```diff
@@ -45,8 +45,8 @@ See [.NET Core 1.x Supported OS Versions](https://github.com/dotnet/core/blob/ma
 * Installing .NET Core via a *.zip* file. This can include build/CI/CD servers.
 
 > [!NOTE]
-> *For Windows 7 and Windows Server 2008 machines only:*
-> Make sure that your Windows installation is up-to-date and includes hotfix [KB2533623](https://support.microsoft.com/help/2533623) installed through Windows Update.
+> *For Windows versions below Windows 10:*
```

Comment:
Should we say this like the following instead?
"For Windows 8.1 and earlier versions, or Windows Server 2012 R2 and earlier versions: